presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1743,4 +1751,15 @@ void PrestoServer::createTaskManager() {
       driverExecutor_.get(), httpSrvCpuExecutor_.get(), spillerExecutor_.get());
 }
 
+void PrestoServer::reportNodeStats(proxygen::ResponseHandler* downstream) {
+  protocol::NodeStats nodeStats;
+
+  // Set overload flags
+  nodeStats.loadMetrics.cpuOverload = cpuOverloaded_;
+  nodeStats.loadMetrics.memoryOverload = memOverloaded_;
+
```

Comment:
We are not exporting these in this PR:
  double cpuUsedPercent = {};
  double memoryUsedInBytes = {};
  int numQueuedDrivers = {};
?

In the following up changes?